src/reducers/index.js

```diff
@@ -0,0 +1,7 @@
+import { combineReducers } from 'redux';
+import bookReducer from './book';
+import filterReducer from './filter';
+
+const reducer = combineReducers({ books: bookReducer, filter: filterReducer });
```

Comment:
**[Optional]**

I think we can have it in one line like this 👇 

`export default combineReducers({ bookStore, filter });`